Libraries/ReactIOS/YellowBox.js

```diff
@@ -186,6 +225,7 @@ class YellowBox extends React.Component {
     super(props, context);
     this.state = {
       inspecting: null,
+      stacktraceVisible: false,
       warningMap: _warningMap,
     };
     this.dismissWarning = warning => {
```

Comment:
@vjeux, @frantic, @davidaurelio, @bestander wow lint is so drunk - sourcemaps? Not sure who owns this stuff these days...